modules/web/styleswitcher.js

```diff
@@ -98,3 +117,23 @@ window.onunload = () => {
 const cookie = readCookie('style');
 const title = cookie || getPreferredStyleSheet();
 setActiveStyleSheet(title);
+/**
```

Comment:
Wrong place for the copyright header.

Reply:
Fixed with https://github.com/ballerinalang/composer/pull/4592